chain/solana/src/runtime/generated.rs

```diff
@@ -0,0 +1,306 @@
+// use diesel::dsl::Asc;
+use graph::runtime::{AscIndexId, AscPtr, AscType, DeterministicHostError, IndexForAscTypeId};
+use graph::semver::Version;
+use graph_runtime_derive::AscType;
+use graph_runtime_wasm::asc_abi::class::{Array, AscString, Uint8Array};
+
+pub(crate) type AscHash = Uint8Array;
+
+#[repr(C)]
+#[derive(AscType)]
+pub(crate) struct AscBlock {
+    pub number: u64,
+    pub previous_block: u64,
+    pub genesis_unix_timestamp: u64,
+    pub clock_unix_timestamp: u64,
+    pub root_num: u64,
+    pub transaction_count: u32,
+    pub version: u32,
+    pub id: AscPtr<Uint8Array>,
+    pub previous_id: AscPtr<Uint8Array>,
+    pub last_entry_hash: AscPtr<Uint8Array>,
+    pub transactions: AscPtr<AscTransactionArray>,
+    pub account_changes_file_ref: AscPtr<AscString>,
+    pub has_split_account_changes: bool,
+    pub _padding: Padding3,
+}
+
+impl AscIndexId for AscBlock {
+    const INDEX_ASC_TYPE_ID: IndexForAscTypeId = IndexForAscTypeId::SolanaBlock;
+}
+
+#[repr(C)]
+#[derive(AscType)]
+pub(crate) struct AscInstruction {
+    pub ordinal: u32,
+    pub parent_ordinal: u32,
+    pub depth: u32,
+    pub program_id: AscPtr<AscHash>,
+    pub account_keys: AscPtr<AscHashArray>,
+    pub data: AscPtr<AscHash>,
+    pub balance_changes: AscPtr<AscBalanceChangeArray>,
+    pub account_changes: AscPtr<AscAccountChangeArray>,
```

Comment:
Let's not push that out at the moment. Let's figure if we can add the `logs` in here.